orchestrator/graphql/types.py

```diff
@@ -132,6 +132,12 @@ class GraphqlFilter:
 }
 
 
+@strawberry.type(description="User permissions on a specific process")
+class FormUserPermissions:
```

Comment:
Nit: the strawberry graphql types are usually suffixed with `Type` :) 
```suggestion
class FormUserPermissionsType:
```